scripts/spcgeonode/geoserver/docker-entrypoint.sh

```diff
@@ -27,8 +27,12 @@ else
         BASEURL="$BASEURL:$HTTP_PORT"
     fi
 fi
+export INTERNAL_OAUTH2_BASEURL="${INTERNAL_OAUTH2_BASEURL:=$BASEURL}"
+export GEONODE_URL="${GEONODE_URL:=$BASEURL}"
```

Comment:
Need 2 extra parameters to define connection for internal and external comunication with the OAuth2 endpoint.